experiments/example/main.go

```diff
@@ -139,11 +139,11 @@ func main() {
 					experiment.ExperimentKey: uid,
 				}
 				// Launching Mutilate Snap session in order to gather metrics on Memcached performance.
-				snap, err := mutilateSnapSession.LaunchSession(mutilateTask, tags)
+				snapTask, err := mutilateSnapSession.LaunchSession(mutilateTask, tags)
```

Comment:
@skonefal - rename to `mutilateSessionHandle` (in other experiments you seem to use "SessionHandle" suffix).